tests/vectors/resolution/longFormResponse.json

```diff
@@ -0,0 +1,43 @@
+{
+  "status": "succeeded",
```

Comment:
I see a status field in this response (and the deactivated example) but not the other ones.

Reply:
@troyronda, this is actually expected, because the is actually the output the `RequestHandler.handleResolveRequest`, others are the output of `DocumentComposer.transformToExternalDocument`